app/controllers/salesforce/admin_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Salesforce
+  class AdminController < ::Admin::AdminController
```

Comment:
I might be wrong but I don't think we're inheriting from this controller any where?

Reply:
No, but I'm using the `authorize` method in this controller for [a link in the locale](https://github.com/discourse/discourse-salesforce/pull/4/files#diff-333c4c8cacc2f84633d4894f2aaddb130a6d74f42dd63199462b3d66278f6ce8R16).

Reply:
ahh icic. Can we add a test for this?